logstream.go

```diff
@@ -73,6 +73,10 @@ func (s *LogStream) findStream() (*cloudwatchlogs.LogStream, error) {
 		return nil, nil
 	}
 
+	if *resp.LogStreams[0].LogStreamName != *s.Stream {
```

Comment:
Does the limit need to be removed so we can check all results in the `DescribeLogStreamsInput` response?